### DISCUSSION TITLE
Canvas cannot be moved with move functionality on left mouse key

### DISCUSSION THREAD

**Discussion by PeterBenak** [cvat-ai/cvat#2903] [Q&A] [answered]
Hi all,

Previously, I could move an image by holding down the left mouse button while creating a polygon object.
Now that I'm creating a polygon object, I can't move the image by holding down the left mouse button.

How can I enable / get this feature in the cvat version

Server version: 1.2
Core version: 3.10.0
Canvas version: 2.2.1
UI version: 1.13.3

I can move the image by holding the mouse wheel button, but it greatly slows down the work.

Thank you.

**Comment by bsekachev** [comment, marked as answer]
Hi @PeterBenak 

The feature was redesigned by customer request. Hold 'Alt' during creating a polygon to move canvas instead of making points.
